components/exercises/chapter-6-oscar/group/MembersGroup.tsx

```diff
@@ -71,7 +71,7 @@ export default function MembersGroup(): JSX.Element {
         <p className='my-2 text-red-500'>Member is already in the list</p>
       )}
       <ul>
-        {members.getMembers().map((value) => (
+        {[...members].map((value) => (
```

Comment:
Shouldn't this use `GroupIterator`?

Reply:
I have refactored using Array.from() to convert the iterator returned by members[Symbol.iterator]() into an array, iterating directly over the iterator.